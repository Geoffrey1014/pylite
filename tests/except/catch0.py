try:
    0/0
except IOError:
    print 'catch IOError'
