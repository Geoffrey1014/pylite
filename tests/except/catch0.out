ZeroDivisionError:
  At catch0.py, line 2, in __init__()
